src/ResourceManager/AnalysisServices/AzureRM.AnalysisServices.psd1

```diff
@@ -9,7 +9,7 @@
 @{  
   
 # Version number of this module.  
-ModuleVersion = '0.0.1'
```

Comment:
This PR is targeting dev branch, it should be targeting release 3.2.0 branch. Please send a new PR targeting the right branch.